PacketDelivery-VC-ReferenceExample/data-consumer/README.md

```diff
@@ -0,0 +1,173 @@
+# Data Service Consumer with verifiable credentials
+
+This describes how to setup the environment of a data service consuming organisation within the i4Trust trusted data space with the example of the fictitious organisations
+
+* Packet Delivery Company being the service provider
+* Happy Pets Inc. being the consuming organisation
+
+Needed for issuing credentials that can be used against the authentication of the service provider's environment are the following
+* [WaltId](https://github.com/walt-id/waltid-ssikit)
+* [Keycloak](https://www.keycloak.org/) with the [VC Issuer plugin](https://github.com/wistefan/keycloak-vc-issuer)
+
+Furthermore are needed
+* VC compatible Wallet for executing the OpenIDConnect flow
+* iSHARE Satellite as trust anchor for sharing trusted parties
+
+![Components](./img/components.png "Components")
+
+Note that there is not an example of a Shop System provided. In the use case example, it is assumed that there are already 
+users registered at Happy Pets and which have placed orders. More precisely, this means that there are already 
+existing delivery order entities within the Context Broker of 
+the [Data Service Provider](../Data-Service-Provider).
+
+Furthermore it is required that there is access to an iSHARE Satellite instance and an iSHARE-compliant Authorisation 
+Registry which stores the user's access policies.
+
+In the following it is assumed that the components will be externally available via the domain `domain.org`. 
+
+## Installation
+
+It is recommended to at least change the "route.host" property of the keycloak chart to adjust to the own test environment.
```

Comment:
You should add a small info(maybe as a box at the beginning) that this will show the openshift version and one should configure the ingress according to https://github.com/bitnami/charts/blob/main/bitnami/keycloak/values.yaml#L529 . "Route" is just available for openshift. The easiest way to get a host there is to not set route.host at all, since it gets a assigned a default route then.